metaflow/plugins/aws/step_functions/step_functions_deployer.py

```diff
@@ -50,6 +50,14 @@ def terminate(self, **kwargs) -> bool:
 
 
 class StepFunctionsDeployedFlow(DeployedFlow):
+    @classmethod
+    def from_deployment(
+        cls, identifier: str, metadata: Optional[str] = None, _="argo-workflows"
```

Comment:
same comment, do we really need the third argument i.e. _="argo-workflows" here?

Reply:
And it should be "step-functions" :)

Reply:
yeah, I didn't mention it because I tilt towards removing this from the underlying implementations..